config/release/build.yml

```diff
@@ -8,17 +9,18 @@ sources:
   docker:
     buildx:
       pull: true
-      noCache: #@ not data.values.image_cache
+      noCache: true
       #! pass kapp_controller_version into Dockerfile.
       rawOptions:
-      - #@ "--build-arg=KCTRL_VER="+data.values.kapp_controller_version
-      #@ if/end data.values.platform != "":
-      - #@ "--platform="+data.values.platform
+      - #@ "--build-arg=KCTRL_VER="+data.values.dev.version
+      #@ if/end data.values.dev.platform != "":
+      - #@ "--platform="+data.values.dev.platform
+      #@ end
 
-#@ if/end data.values.push_images:
+#@ if/end data.values.dev.push_images:
```

Comment:
We can remove this check from here now as we have separate build.yml for dev and release as for release we always want to push the image.